integration/src/test/scala/com/spotify/scio/elasticsearch/ElasticsearchIOBehavior.scala

```diff
@@ -103,8 +102,7 @@ trait ElasticsearchIOBehavior extends Eventually with ForAllTestContainer { this
       val host = new HttpHost(container.host, container.mappedPort(9200))
       val esOptions = ElasticsearchOptions(
         nodes = Seq(host),
-        usernameAndPassword = Some((Username, Password)),
-        mapperFactory = createScalaMapper
+        usernameAndPassword = Some((Username, Password))
```

Comment:
`createScalaMapper` is now unused and can be removed. You'll be able to remove the unused libs from the project too

Reply:
It's still used by the `RestClientTransport`

Reply:
my bad